lib/middleware/express-extension-to-accept.js

```diff
@@ -16,7 +16,7 @@ module.exports = function (formats) {
       throw new Error('Invalid format.')
   })
 
-  var regexp = new RegExp('\.(' + formats.join('|') + ')$', 'i')
+  var regexp = new RegExp('.(' + formats.join('|') + ')$', 'i')
```

Comment:
Hi, I like this PR's nice tidying up and I've tested your branch a bit without any problems.
I've noticed one old slight issue here:
I think that escaping of dot should be done here, but I guess we need `\\.` correctly.
Without double backslash the dot has the meaning of any character so that this URL is going to be valid:
`https://localhost/api/v1/entriesXjson`
But we want only `https://localhost/api/v1/entries.json` to be valid.
Maybe you could add a fix of this here, too?

Reply:
Good catch, thanks for calling this out! I updated the regex to use double backslash `\\`. I also added tests for a valid case and an invalid case as a sanity check.